documentation/ProjectReference-Protocol.md

```diff
@@ -51,7 +51,7 @@ These targets should exist in a project to be compatible with the common targets
 
 These targets are all defined in `Microsoft.Common.targets` and are defined in Microsoft SDKs. You should only have to implement them yourself if you require custom behavior or are authoring a project that doesn't import the common targets.
 
-If implementing a project with an “outer” (determine what properties to pass to the real build) and “inner” (fully specified) build, only `GetTargetFrameworkProperties` is required in the “outer” build. The other targets listed can be “inner” build only.
+If implementing a project with an “outer” (determine what properties to pass to the real build) and “inner” (fully specified) build, only `GetTargetFrameworkProperties` and `GetTargetFrameworks` are required in the “outer” build. The other targets listed can be “inner” build only.
```

Comment:
Please verify this change. My understanding is that `GetTargetFrameworks` is also part of the outer build, and that this sentence was not updated when the docs for the target were added below.

Reply:
I think `GetTargetFrameworkProperties` doesn't exist anymore? @dplaisted correct me if I'm wrong, but it looks like `GetTargetFrameworkProperties` may have been renamed to `GetTargetFrameworks`. I grepped through the sdk repo and couldn't find a target with that name. Here's the comment about this target in common targets:

```
  <!--
    ============================================================
                                    GetTargetFrameworkProperties

    Overrridden by cross-targeting projects to return the set of
    properties (in the form "key1=value1;...keyN=valueN") needed
    to build it with the best target for the referring project's
    target framework.

    The referring project's $(TargetFrameworkMoniker) is passed
    in as $(ReferringTargetFramework)
  -->
  <Target Name="GetTargetFrameworkProperties" />
```

This might just be dead code

Reply:
yep https://github.com/dotnet/sdk/pull/15027

Reply:
Yeah, so you're thinking this right?

```suggestion
If implementing a project with an “outer” (determine what properties to pass to the real build) and “inner” (fully specified) build, only `GetTargetFrameworks` are required in the “outer” build. The other targets listed can be “inner” build only.
```